Controller/RouterController.php

```diff
@@ -58,4 +60,30 @@ public function asset(Request $request): Response
 
         return $this->processor->getResponse($request, $data['hash'], $data['config'], $data['filename']);
     }
+
+    public function makeResponse(Request $request): Response
+    {
+        $result = $this->handler->handle($request);
+        $json = $this->templating->render($result['template'], $result['context']);
+
+        $data = \json_decode($json, true);
+
+        if ($data === false) {
+            throw new \RuntimeException('Failed to decode JSON');
```

Comment:
Th error message should explain that a json is expected with at least a content field string